diff -Naur -p1 6.12.5-xxx/drivers/net/ethernet/intel/ixgbe/ixgbe_phy.c 6.12.5-qlt/drivers/net/ethernet/intel/ixgbe/ixgbe_phy.c
--- 6.12.5-xxx/drivers/net/ethernet/intel/ixgbe/ixgbe_phy.c	2025-02-23 12:08:37.590264637 +0200
+++ 6.12.5-qlt/drivers/net/ethernet/intel/ixgbe/ixgbe_phy.c	2025-02-23 11:54:28.120210919 +0200
@@ -1742,2 +1742,5 @@ int ixgbe_identify_sfp_module_generic(st
 			break;
+		case IXGBE_SFF_VENDOR_OUI_MENARA:
+			hw->phy.type = ixgbe_phy_sfp_menara;
+			break;	
 		default:
@@ -1791,9 +1794,17 @@ int ixgbe_identify_sfp_module_generic(st
 			return 0;
-		if (hw->allow_unsupported_sfp) {
-			e_warn(drv, "WARNING: Intel (R) Network Connections are quality tested using Intel (R) Ethernet Optics.  Using untested modules is not supported and may cause unstable operation or damage to the module or the adapter.  Intel Corporation is not responsible for any harm caused by using untested modules.\n");
+		if (hw->phy.type == ixgbe_phy_sfp_menara) {
+			e_warn(drv, "WARNING: Menara SFP detected\n");
 			return 0;
 		}
-		hw_dbg(hw, "SFP+ module not supported\n");
-		hw->phy.type = ixgbe_phy_sfp_unsupported;
-		return -EOPNOTSUPP;
+		else{
+			hw_dbg(hw, "SFP+ module not supported\n");
+			hw->phy.type = ixgbe_phy_sfp_unsupported;
+			if (hw->allow_unsupported_sfp) {
+				e_warn(drv, "WARNING: Intel (R) Network Connections are quality tested using Intel (R) Ethernet Optics.  Using untested modules is not supported and may cause unstable operation or damage to the module or the adapter.  Intel Corporation is not responsible for any harm caused by using untested modules.\n");
+				return 0;
+			}
+			else{
+				return -EOPNOTSUPP;
+			}
+		}
 	}
diff -Naur -p1 6.12.5-xxx/drivers/net/ethernet/intel/ixgbe/ixgbe_phy.h 6.12.5-qlt/drivers/net/ethernet/intel/ixgbe/ixgbe_phy.h
--- 6.12.5-xxx/drivers/net/ethernet/intel/ixgbe/ixgbe_phy.h	2025-02-23 12:08:37.590264637 +0200
+++ 6.12.5-qlt/drivers/net/ethernet/intel/ixgbe/ixgbe_phy.h	2025-02-23 11:54:28.120210919 +0200
@@ -104,2 +104,3 @@
 #define IXGBE_SFF_VENDOR_OUI_INTEL    0x001B2100
+#define IXGBE_SFF_VENDOR_OUI_MENARA   0x004D4E00
 
diff -Naur -p1 6.12.5-xxx/drivers/net/ethernet/intel/ixgbe/ixgbe_type.h 6.12.5-qlt/drivers/net/ethernet/intel/ixgbe/ixgbe_type.h
--- 6.12.5-xxx/drivers/net/ethernet/intel/ixgbe/ixgbe_type.h	2025-02-23 12:08:37.591264637 +0200
+++ 6.12.5-qlt/drivers/net/ethernet/intel/ixgbe/ixgbe_type.h	2025-02-23 11:54:28.121210919 +0200
@@ -3172,2 +3172,3 @@ enum ixgbe_phy_type {
 	ixgbe_phy_sfp_intel,
+	ixgbe_phy_sfp_menara,
 	ixgbe_phy_qsfp_passive_unknown,
