diff -ur 6.12.5-xxx/drivers/net/ethernet/mellanox/mlx5/core/en/rqt.h 6.12.5-qlt/drivers/net/ethernet/mellanox/mlx5/core/en/rqt.h
--- 6.12.5-xxx/drivers/net/ethernet/mellanox/mlx5/core/en/rqt.h 2025-01-19 09:03:57.585225084 +0200
+++ 6.12.5-qlt/drivers/net/ethernet/mellanox/mlx5/core/en/rqt.h 2024-12-26 20:13:36.441337103 +0200
@@ -6,7 +6,7 @@

 #include <linux/kernel.h>

-#define MLX5E_INDIR_MIN_RQT_SIZE (BIT(8))
+#define MLX5E_INDIR_MIN_RQT_SIZE (BIT(11))

 struct mlx5_core_dev;


